Cycle_Sort/Cycle_Sort.cs

```diff
@@ -0,0 +1,65 @@
+// C# program for cycle sort 
+using System; 
+  
+class CycleSort { 
+      
+    // Function to implement Cycle sort 
+    public static void cycleSort(int[] arr, int n) 
+    {
+        int writes = 0; 
+        // traverse through array elements and find the right position  
+        for (int cycle_start = 0; cycle_start <= n - 2; cycle_start++) 
+        {  
+            int item = arr[cycle_start]; 
+            int pos = cycle_start; 
+            for (int i = cycle_start + 1; i < n; i++) 
+                if (arr[i] < item) 
+                    pos++; 
+                    
+            if (pos == cycle_start) 
+                continue; 
+  
+            while (item == arr[pos]) 
+                pos += 1; 
+  
+            if (pos != cycle_start) { 
+                int temp = item; 
+                item = arr[pos]; 
+                arr[pos] = temp; 
+                writes++; 
+            } 
+  
+            while (pos != cycle_start) { 
+                pos = cycle_start; 
+   
+                for (int i = cycle_start + 1; i < n; i++) 
+                    if (arr[i] < item) 
+                        pos += 1; 
+  
+                while (item == arr[pos]) 
+                    pos += 1; 
+  
+                if (item != arr[pos]) { 
+                    int temp = item; 
+                    item = arr[pos]; 
+                    arr[pos] = temp; 
+                    writes++; 
```

Comment:
correct the indentation

Reply:
Mam there is 4 space indentation right . Pls tell where do I go wrong

Reply:
>  if (pos == cycle_start) 
>                 continue; 
> 
>             while (item == arr[pos]) 
>                 pos += 1; 
> 
>             if (pos != cycle_start) { 
>                 int temp = item; 
>                 item = arr[pos]; 
>                 arr[pos] = temp; 
>                 writes++; 

these if statements and while loop is come inside the for loop nah? you didn't keep indentation in it

Reply:
@asha15  Mam those if and while loop don't come inside the for loop .I have added more comments . Actually the for , if , while statements come inside while(pos != cycle_start) Pls check it out..

Reply:
@Harikrishnan6336 great, now it looks good. 
please all your commits into one and we are good to merge

Reply:
Mam  ,I made my final commit too

Reply:
please squash all your commits we are good to merge

Reply:
@asha15  How do we squash commits?

Reply:
@asha15  Mam I find out through stackoverflow that we can squash commits only with the use of git in command prompt. Since I am not so good at git , can I close this PR and create a new one

Reply:
@Harikrishnan6336 no you can do it, i'll help you
reach me on slack

Reply:
@asha15  Mam I was able to do it .

Reply:
@Harikrishnan6336 You are awesome. We'll merge the PR.
congratulations